tomatic/callinfo.py

```diff
@@ -134,6 +134,7 @@ def partnerInfo(self, partner_data):
                 ([self.anonymize(partner_data.www_mobile)] if partner_data.www_mobile else [])
             ),
             comment=partner_data.comment or '',
+            unsubscribed=self.config.partner_category_id not in partner_data.category_id,
```

Comment:
I would suggest to add more semantics to the name (ie. member_category_id) as there are many categories that we could check. `partner_category_id` does not explain what are we doing here sinc all persons are partners in erp.

Also it is a weird parameter to configure since one has to check which is the id for that category in the erp instance. A better approach would be using a semantic id for the category and let it be just a constant, not a configuration parameter.